katello/katello-backup

```diff
@@ -44,14 +44,22 @@ optparse = OptionParser.new do |opts|
   @dir = ARGV[0].dup
 end
 
+def run_cmd(command)
+  `#{command}`
+  if $?.exitstatus != 0
```

Comment:
If you've got error snippets, I can make it smarter.  I'd rather fix the check than defeat it.